broker-cli/commands/orderbook.js

```diff
@@ -162,19 +291,51 @@ async function orderbook (args, opts, logger) {
 };
 
 /**
- * Takes in an window width and returns object with appropriate table widths
- * for the orderbook
- * @param {Integer} window width
- * @returns {Object} with mainTableWidth and innerTableWidth
+ * Takes a number formatted as a string and applies coloring so all digits following
+ * the last significant digit are grayed out
+ * @param {string} text
+ * @param {string} type - represents whether to format as a bid, ask, or depth
+ * @returns {string}
  */
+function formatText (text, type) {
+  let lastZeroIndex = -1
+  for (let i = 0; i < text.length; i++) {
```

Comment:
you might pull this into its own function, like `findLastZero`. Also consider using a function like: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf which may be helpful if you reverse the string first, for example.

Reply:
Pulled this into another function.

I think if we reverse the string then look for the index of the first occurrence of a non-`0`, then do a `lastIndexOf` `0` and find the unreversed index, it might be less clear what's going on with roughly the same length function. You okay keeping it as-is in a separate function?